android/app/src/main/res/values-ca/strings.xml

```diff
@@ -377,6 +377,8 @@
 	<string name="search_not_found_query">Proveu de canviar els criteris de cerca.</string>
 	<string name="search_history_title">Historial de cerca</string>
 	<string name="search_history_text">Mostra les cerques recents.</string>
+	<!-- Preference description -->
+	<string name="pref_search_history_summary">Manteniu l\'historial de cerques</string>
```

Comment:
Let's please change the conjugation here to "Mantingues" instead of "Manteniu". We use "vós" only when the command is addressed to the user; when it's addressed to the software, Softcatalà Style Guide says to use "tu".